.github/ISSUE_TEMPLATE/bug-report.md

```
@@ -31,4 +31,6 @@ If applicable, add screenshots to help explain your problem.
 Add any other context or screenshots about the feature request here.
 
 **Potential internship candidates**
-Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359
+
+Please read this if you are planning to apply for a Palisadoes Foundation internship 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)